### DISCUSSION TITLE
Large construction site teams.

### DISCUSSION THREAD

**Discussion by TropicalStoic** [ldtteam/minecolonies#10334] [Ideas]
Add the option to assign 2 or 3 builders to a single construction project to build it faster while maintaining the game's difficulty and progression logic. It would still require the player to build 2 or 3 builder huts beforehand. Sometimes I want to rush a specific medium/large construction that takes too long for a single builder to finish, even if I have more builders available, I can't use them simultaneously. It's common sense that multiple workers can and do work together on construction sites.

If game balance is a concern, a certain level of Builder's Hut or Technology research may be required to unlock large construction crews.

**Comment by Talyda** [comment]
Just to copy paste the relevant bit of the response we give on discord:

(...)
However, it is not possible and will not be implemented to have multiple builders working on the same building at the same time due to technical issues regarding fluids and block states.
